modules/reissue-internal-certificates-secured-clusters.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * configuration/reissue-internal-certificates.adoc
+:_module-type: PROCEDURE
+[id="reissue-internal-certificates-secured-cluster_{context}"]
+= Reissuing internal certificates for Secured Clusters using init bundles
+
+Secured clusters contain the Collector, Sensor, and Admission Control components. These components use a built-in server certificate for authentication when communicating with other {product-title} components.
+
+The RHACS portal shows an information banner when the Central certificate is about to expire.
+
+[NOTE]
+====
+The information banner only appears 15 days before the certificate expiry date.
+====
+
+.Prerequisites
+
+* To reissue certificates, you must have `write` permission for the `ServiceIdentity` resource.
+
+[IMPORTANT]
+====
+Store this bundle securely because it contains secrets.
+You can use the same bundle on multiple secured clusters.
+====
+
+.Procedure
+
+To generate an init bundle using the {product-title-short} portal:
+
+. Select *Platform Configuration* -> *Clusters*.
+. Click *Manage Tokens*.
+. Navigate to the *Authentication Tokens* section, and click *Cluster Init Bundle*.
+. Click *Generate bundle*.
+. Enter a name for the cluster init bundle and click *Generate*.
+. Click *Download Kubernetes secrets file* to download the generated bundle.
```

Comment:
Just a suggestion, I think it works either way but I'm biased towards goal-first language :)

```suggestion
. To download the generated bundle, click *Download Kubernetes secrets file*.
```